infra/partner-vm-testnet/network.tf

```diff
@@ -39,4 +39,19 @@ module "vpc" {
         ]
       },
      ]
+}
+
+resource "google_compute_router" "router" {
+  name    = "default"
+  network = var.network
+  project = var.project_id
+  region  = var.region
+}
+
+resource "google_compute_router_nat" "nat" {
```

Comment:
wait isn't this already merged?

Reply:
Yeah, that's odd that it shows up here actually 🤔

Reply:
Fixed, I had some branch craziness going on 😅